clients/src/main/java/org/apache/kafka/clients/admin/AdminClientConfig.java

```diff
@@ -193,7 +195,7 @@ public AdminClientConfig(Map<?, ?> props) {
     }
 
     protected AdminClientConfig(Map<?, ?> props, boolean doLog) {
-        super(CONFIG, props, doLog);
+        super(CONFIG, props, doLog, ENABLE_INDIRECT_RESOLUTION);
```

Comment:
I think this change and changes to other classes which don't use config.providers was required because a public constructor was removed. We need to retain existing public constructors. So perhaps we could revert changes to classes which don't use config providers?